packages/edge-config/src/index.ts

```diff
@@ -113,6 +112,70 @@ const getPrivateEdgeConfig = trace(
   },
 );
 
+/**
+ *
+ */
+function createGetInMemoryEdgeConfig(
+  useDevelopmentCache: boolean,
+  connection: Connection,
+  headers: Record<string, string>,
+): () => Promise<EmbeddedEdgeConfig | null> {
+  let embeddedEdgeConfigPromise: Promise<EmbeddedEdgeConfig | null> | null =
```

Comment:
A description of embeddedEdgeConfigPromise and lastRequest would help make the code below more readable. At the moment I get it roughly but it's hard to follow some of the nuance